cmd/core-metadata/main.go

```diff
@@ -23,44 +23,55 @@ import (
 
 	"github.com/edgexfoundry/edgex-go/core/metadata"
 	logger "github.com/edgexfoundry/edgex-go/support/logging-client"
+	"fmt"
 )
 
 const (
-	CONFIG = "res/configuration.json"
+	configFile = "res/configuration.json"
 )
 
-func main() {
-	var loggingClient = logger.NewClient(metadata.METADATASERVICENAME, "")
+var loggingClient logger.LoggingClient
 
+func main() {
 	// Load configuration data
-	configuration, err := readConfigurationFile(CONFIG)
+	configuration, err := readConfigurationFile(configFile)
 	if err != nil {
-		loggingClient.Error("Could not read configuration file(" + CONFIG + "): " + err.Error())
+		loggingClient = logger.NewClient(metadata.METADATASERVICENAME, false, "")
+		loggingClient.Error("Could not read configuration file(" + configFile + "): " + err.Error())
 		return
 	}
 
-	// Update logging based on configuration
-	loggingClient.RemoteUrl = configuration.LoggingRemoteURL
-	loggingClient.LogFilePath = configuration.LoggingFile
+	logTarget := setLoggingTarget(*configuration)
+	// Create Logger (Default Parameters)
+	loggingClient = logger.NewClient(configuration.ApplicationName, configuration.EnableRemoteLogging, logTarget)
 
 	metadata.Start(*configuration, loggingClient)
 }
 
 // Read the configuration file and
 func readConfigurationFile(path string) (*metadata.ConfigurationStruct, error) {
 	var configuration metadata.ConfigurationStruct
-
 	// Read the configuration file
 	contents, err := ioutil.ReadFile(path)
 	if err != nil {
+		fmt.Println("Error reading configuration file: " + err.Error())
 		return nil, err
 	}
 
 	// Decode the configuration as JSON
 	err = json.Unmarshal(contents, &configuration)
 	if err != nil {
+		fmt.Println("Error reading configuration file: " + err.Error())
```

Comment:
Remove printf. It is already logged in line 40

Reply:
Actually, no. This is different but I will change the log message accordingly. This is actually an error with parsing the JSON after the file has been found.